tests/pl/person_HassGetState.yaml

```diff
@@ -0,0 +1,77 @@
+language: pl
+tests:
+  - sentences:
+      - "Gdzie jest Witold?"
+    intent:
+      name: HassGetState
+      slots:
+        domain: person
+        name: "Witold"
+    response: "Witold jest w dom"
+
+  - sentences:
+      - "Gdzie jest Maria?"
+    intent:
+      name: HassGetState
+      slots:
+        domain: person
+        name: "Maria"
+    response: "Maria jest w praca"
+
+  - sentences:
+      - "Czy Maria jest w dom?"
```

Comment:
you challenged me on the exact same issue here https://github.com/home-assistant/intents/pull/1993#discussion_r1503108221 ;)

Reply:
Not really 😀 The `dom` is the value of key `state` (we can't use few names for state), but in your case the `Poznań` or `Warszawa` is just value for key name - In this case you can conjugate the name.

Reply:
I reality you can actually create aliases to specified entity names, which you can't do to the state value itself.